qdata/parser.py

```diff
@@ -89,7 +94,7 @@ def statement(self, *args):
             # opaque <id> <idlist>; or
             # opaque <id> () <idlist>; or
             # opaque <id> (<idlist>) <idlist>;
-            decl = self.qdecl(Declaration, opaque=True, *args[1:])
+            decl = self.qdecl(Declaration, *args[1:], opaque=True)
```

Comment:
was this a bug? how did you uncover?

Reply:
I can't remember. Let me revert it just to be safe.
```suggestion
            decl = self.qdecl(Declaration, opaque=True, *args[1:])
```